sdk/cognitivelanguage/ai-language-text/swagger/README.md

```diff
@@ -12,9 +12,9 @@ generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../
 source-code-folder-path: ./src/generated
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/dev/cognitiveservices/data-plane/Language/analyzetext.json
```

Comment:
They are in the process of moving this, but in your autorest configuration you should always use a commit. Branch and tags can move, and branches most definitely will move.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/23208/commits/538c4c6499ae22ebebf39ae2ad4fb21df682592f